README.md

```diff
@@ -96,7 +86,6 @@ The key subdirectories are:
  - For the main distributed CT Log itself:
    - `cpp/log`: Main distributed CT Log implementation.
    - `cpp/merkletree`: Merkle tree implementation.
-   - `cpp/server`: Top-level code for server implementations.
    - `cpp/monitoring`: Code to export operation statistics from CT Log.
  - The [CT mirror Log](docs/MirrorLog.md) implementation also uses:
    - `cpp/fetcher`: Code to fetch entries from another Log
```

Comment:
`cpp/fetcher/` looks to get removed here; should it be? (I.e. does anything else use it?)  If so, drop this line too.

Reply:
Removed!